include/sound/sof/control.h

```diff
@@ -99,6 +99,9 @@ struct sof_ipc_ctrl_data {
 	struct sof_ipc_reply rhdr;
 	uint32_t comp_id;
 
+	/* for large messages sent in parts */
+	uint32_t msg_id;
+
```

Comment:
this will break the ABI, you need to use one of the reserved fields?

Reply:
yes, in the discussion I asked could I use the reserved fields in sof_abi_hdr member. I think sof_ipc_ctrl_data doesn't have those...?